pkg/helm/charts_test.go

```diff
@@ -40,18 +26,33 @@ func TestAddRepos(t *testing.T) {
 }
 
 func TestAdd(t *testing.T) {
-	c, err := InitChartfile(filepath.Join(t.TempDir(), Filename))
+	tempDir := t.TempDir()
```

Comment:
The tests now depend on actual helm pulling the prometheus chart. Since the `helm pull` behavior is a bit unusual, I believe that a full test is better than mocking the Helm process